middleware.ts

```diff
@@ -1,5 +1,5 @@
 export { auth as middleware } from './auth'
 
 export const config = {
-  matcher: ['/', '/api/chat']
```

Comment:
This is likely unintended. Since `/api/chat` has an inline auth (`await auth()`), the Middleware would be redundant.